spec/features/locations/bulk_upload_spec.rb

```diff
@@ -1,14 +1,28 @@
 describe "Bulk upload locations and IPs", type: :feature do
   let(:user) { create(:user, :with_organisation) }
   let(:organisation) { user.organisations.first }
+  let!(:another_administrator) { create(:user, organisations: [user.organisations.first]) }
+
+  context "when there is only one administrator for the organisation" do
+    before do
+      user.organisations.first.users.delete(another_administrator)
```

Comment:
Would it not be better to not create the 'another_administrator'  in the first place?